drivers/storage/portworx/component/telemetry.go

```diff
@@ -281,6 +289,16 @@ func (t *telemetry) getCollectorDeployment(
 						{
 							Name:  "collector",
 							Image: collectorImage,
+							Resources: v1.ResourceRequirements{
+								Requests: v1.ResourceList{
+									v1.ResourceMemory: resource.MustParse(defaultCollectorMemoryRequest),
+									v1.ResourceCPU:    cpuQuantity,
+								},
+								Limits: v1.ResourceList{
```

Comment:
don't need cpu limit here?

Reply:
probably no customer requested it, other components also do not have it.